README.md

```diff
@@ -23,7 +23,7 @@ store.dispatch({ type: "INCREMENT" });
 console.log(store.getState().count); // => 1
 ```
 
-`create-state` is a lightweight implementation of a [Redux][] store without advanced concepts like middleware.
+`create-state` is a lightweight implementation of a [Redux](https://redux.js.org/introduction) store without advanced concepts like middleware.
```

Comment:
This works as expected and is using [reference-style links](https://daringfireball.net/projects/markdown/syntax#link). 🙂 

```suggestion
`create-state` is a lightweight implementation of a [Redux][] store without advanced concepts like middleware.
```